### ISSUE TITLE
[api-extractor] First @link within @param descriptions has the preceding space trimmed

### ISSUE THREAD

**Issue by AlexJerabek** [Microsoft/web-build-tools#726]
Given a line like
`@param handler The event handler function to add, whose only parameter is of type {@link Office.BindingDataChangedEventArgs} or {@link Office.BindingSelectionChangedEventArgs}.`

The resulting json output is:
```
 "handler": {
                  "name": "handler",
                  "description": [
                    {
                      "kind": "text",
                      "text": "The event handler function to add, whose only parameter is of type"
                    },
                    {
                      "kind": "api-link",
                      "elements": [
                        {
                          "kind": "text",
                          "text": "Office.BindingDataChangedEventArgs"
                        }
                      ],
[…]
```
This eventually results in API-Documenter's yml looking like this:
```
The event handler function to add, whose only parameter is of
            type[Office.BindingDataChangedEventArgs](xref:office.Office.BindingDataChangedEventArgs) or
            [Office.BindingSelectionChangedEventArgs](xref:office.Office.BindingSelectionChangedEventArgs)
```
Note the missing space between type and the first link (though not the second).

**Comment by pgonzal** [comment]
This will be fixed using the helper from TSDoc [PR #61](https://github.com/Microsoft/tsdoc/pull/61).

**Comment by pgonzal** [comment]
This should have been fixed in API Extractor 6.  (If not, please reopen this issue.)
